eval.py

```diff
@@ -15,9 +15,11 @@ def main():
     parser.add_argument('--inferred', required=True)
     parser.add_argument('--output')
     parser.add_argument('--logdir')
+    parser.add_argument('--evaluate_beams_individually', action='store_true')
```

Comment:
```suggestion
    parser.add_argument('--evaluate-beams-individually', action='store_true')
```
to match --config-args